src/browser/components/SettingsPage.jsx

```diff
@@ -879,6 +903,29 @@ export default class SettingsPage extends React.Component {
       </Checkbox>
     );
 
+    options.push(
+      <div>
+        <div>{'Download Location'}</div>
+        <input
+          style={settingsPage.downloadLocationInput}
+          key='inputDownloadLocation'
+          id='inputDownloadLocation'
+          ref='downloadLocation'
```

Comment:
can we better use `createRef` instead? eventually we'll have to migrate any string references as well, but we can try and prevent adding new ones